### PR TITLE
docs(bodiless-documentation): github labeling

### PR DESCRIPTION
**Pull request by hvanyo**
## Changes
Updated Contributing documentation to reflect github labeling.

### CODE REVIEW

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 1]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+- "bug": issue with current behavior.
+- "enhancement": feature enhancement.
+- "documentation": documentation change/request/addition.
+- "testing enhancement": automation story enhancements.
+- "windows": the bug is specific to Windows platform.
+
+Additonal labels that can be used as needed:
+- "duplicate": duplicate of another issue or bug.
+- "invalid": This doesn't seem right.
+- "Further information is requested": Submitter should response with more information/clarifications.
```
Comment (asiarheyeu) [inline packages/bodiless-documentation/doc/Development/Contributing.md:65]: Should we rename "Further information is requested" to "information is requested"?

Comment (hvanyo) [reply packages/bodiless-documentation/doc/Development/Contributing.md:65]: I checked and label is "question" with description: 
Further information is requested.  https://github.com/johnsonandjohnson/Bodiless-JS/labels
Does this work for you?

Comment (hvanyo) [reply packages/bodiless-documentation/doc/Development/Contributing.md]: changed label to "question"

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 2]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+  the core team. Pull requests are welcome.  
+
+In addition each label will be set on the issue to help with additional categorizing. 
+- "bug": issue with current behavior.
+- "enhancement": feature enhancement.
+- "documentation": documentation change/request/addition.
+- "testing enhancement": automation story enhancements.
+- "windows": the bug is specific to Windows platform.
+
+Additonal labels that can be used as needed:
```
Comment (asiarheyeu) [inline packages/bodiless-documentation/doc/Development/Contributing.md:62]: Additonal -> Additional

Comment (hvanyo) [reply packages/bodiless-documentation/doc/Development/Contributing.md]: fixed sp

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 3]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+- "bug": issue with current behavior.
+- "enhancement": feature enhancement.
+- "documentation": documentation change/request/addition.
+- "testing enhancement": automation story enhancements.
+- "windows": the bug is specific to Windows platform.
+
+Additional labels that can be used as needed:
+- "duplicate": duplicate of another issue or bug.
+- "invalid": This doesn't seem right.
+- "question": Further information is requested. Submitter should response with more information/clarifications.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:65]:
```suggestion
- "question": Further information is requested. Submitter should respond with more information/clarifications.
```

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 4]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+
+In addition each label will be set on the issue to help with additional categorizing. 
+- "bug": issue with current behavior.
+- "enhancement": feature enhancement.
+- "documentation": documentation change/request/addition.
+- "testing enhancement": automation story enhancements.
+- "windows": the bug is specific to Windows platform.
+
+Additional labels that can be used as needed:
+- "duplicate": duplicate of another issue or bug.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:63]:
```suggestion
- "duplicate": Duplicate of another issue or bug.
```

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 5]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
 - "priority": The issue is accepted for development and will be prioritized by the core team.  Pull
   requests are welcome, but the issue will likely be assigned a developer from the core team and be
   included in an upcoming sprint.
+- "help wanted": The issue is accepted for develop but it is NOT prioritized by
+  the core team. Pull requests are welcome.  
+
+In addition each label will be set on the issue to help with additional categorizing. 
+- "bug": issue with current behavior.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:56]:
```suggestion
- "bug": Issue with current behavior.
```

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 6]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+- "enhancement": feature enhancement.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:57]:
```suggestion
- "enhancement": Feature enhancement.
```

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 7]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+- "documentation": documentation change/request/addition.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:58]:
```suggestion
- "documentation": Documentation change/request/addition.
```

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 8]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+- "testing enhancement": automation story enhancements.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:59]:
```suggestion
- "testing enhancement": Automation story enhancements.
```

File: packages/bodiless-documentation/doc/Development/Contributing.md [hunk 9]
```diff
@@ -49,6 +49,21 @@ of the following dispositions:
...
+- "windows": the bug is specific to Windows platform.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Contributing.md:60]:
```suggestion
- "windows": Bug specific to Windows platform.
```

**Review by asiarheyeu — APPROVED** [review]

**Pull request merged**